### DISCUSSION TITLE
How to configure targerUrl when session expired(both lDP and mod_auth_openidc)

### DISCUSSION THREAD

**Discussion by empty117** [OpenIDC/mod_auth_openidc#1094] [General]
In logout, we can configure ?logout=<targetUrl>, so that after logout successful, it can redirect to targetUrl.

However in case session expired(both lDP and mod_auth_openidc), is there any way to configure targetUrl, so that we can customized the redirection whenever expire happens?

**Comment by zandbelt** [comment]
when the session is expired, the action configured in `OIDCUnAuthAction` will be executed, see: https://github.com/OpenIDC/mod_auth_openidc/blob/v2.4.14.2/auth_openidc.conf#L900-L957 ; if you don't want to redirect for authentication you can use e.g. a 403
